File: PRAK404-2410817310009-Muhammad_Ghazi_Rakhmadi.c
#include <stdio.h>

int main() {
    int pilihan;
    float nilai1, nilai2, hasil;

    do {
        printf("Pilihan program:\n");
        printf("1. Penjumlahan\n");
        printf("2. Pengurangan\n");
        printf("3. Perkalian\n");
        printf("4. Pembagian\n");
        printf("5. Exit\n");
        printf("Masukkan Pilihan: ");
        scanf("%d", &pilihan);

        if (pilihan >= 1 && pilihan <= 4) {
            printf("Masukkan nilai pertama: ");
            scanf("%f", &nilai1);
            printf("Masukkan nilai kedua: ");
            scanf("%f", &nilai2);

            switch (pilihan) {
                case 1:
                    hasil = nilai1 + nilai2;
                    printf("Hasil: %.2f\n", hasil);
                    break;
                case 2:
                    hasil = nilai1 - nilai2;
                    printf("Hasil: %.2f\n", hasil);
                    break;
                case 3:
                    hasil = nilai1 * nilai2;
                    printf("Hasil perkalian antara %.2f dengan %.2f adalah %.2f\n\n", nilai1, nilai2, hasil);
                    break;
                case 4:
                    if (nilai2 != 0) {
                        hasil = nilai1 / nilai2;
                        printf("Hasil: %.2f\n", hasil);
                    } else {
                        printf("Error: Pembagian dengan nol tidak diperbolehkan.\n\n");
                    }
                    break;
            }
        } else if (pilihan != 5) {
            printf("Input anda salah, silahkan coba lagi\n\n");
        }
    } while (pilihan != 5);
    printf("Terimakasih, telah menggunakan kalkulator Muhammad Ghazi Rakhmadi\n");
}

File: PRAK404-2410817310009-Muhammad_Ghazi_Rakhmadi.py
while True:
    print("Pilihan program:")
    print("1. Penjumlahan")
    print("2. Pengurangan")
    print("3. Perkalian")
    print("4. Pembagian")
    print("5. Exit")

    pilihan = int(input("Masukkan Pilihan: "))
    if pilihan == 5:
        print("Terimakasih, telah menggunakan kalkulator Muhammad Ghazi Rakhmadi")
        break  

    if pilihan in [1, 2, 3, 4]:
        nilai1 = float(input("Masukkan nilai pertama: "))
        nilai2 = float(input("Masukkan nilai kedua: "))

        if pilihan == 1:  
            hasil = nilai1 + nilai2
            print(f"Hasil penjumlahan antara {nilai1:.2f} dengan {nilai2:.2f} adalah {hasil:.2f}")
        elif pilihan == 2: 
            hasil = nilai1 - nilai2
            print(f"Hasil pengurangan antara {nilai1:.2f} dengan {nilai2:.2f} adalah {hasil:.2f}")
        elif pilihan == 3:  
            hasil = nilai1 * nilai2
            print(f"Hasil perkalian antara {nilai1:.2f} dengan {nilai2:.2f} adalah {hasil:.2f}\n")
        elif pilihan == 4:  
            if nilai2 != 0:
                hasil = nilai1 / nilai2
                print(f"Hasil pembagian antara {nilai1:.2f} dengan {nilai2:.2f} adalah {hasil:.2f}")
            else:
                print("Error: Pembagian dengan nol tidak diperbolehkan.")
    else:
        print("Input anda salah, silahkan coba lagi\n")